app/components/document_title_component.rb

```diff
@@ -11,4 +11,10 @@ def object_type_label
   def object_type_class
     object_type_label.tr(' ', '-')
   end
+
+  def version_banner
+    return 'You are viewing the latest version.' if @presenter.head_user_version_view?
+
+    'You are viewing an older version.' if @presenter.user_version_view?
+  end
```

Comment:
What would you think about this method living in the presenter itself? Currently, how and whether the banner is presented depends entirely on presenter-based concerns, so it seems like we could make a case for it.

Reply:
Since it is only used in one place, I'd prefer to see the logic closer to where the rendering is performed.